os/arch/arm/src/armv7-a/arm_dataabort.c

```diff
@@ -78,6 +78,8 @@ uint32_t system_exception_location;
 
 static inline void print_dataabort_detail(uint32_t *regs, uint32_t dfar, uint32_t dfsr)
 {
+	/* Abort log must always start at a new line.*/
+	lldbg_noarg("\n");
 	_alert("#########################################################################\n");
```

Comment:
Let me suggest below
```
_alert("\n###################
```

Reply:
current one is the same as previous. just location is different. But it's not important.

Reply:
as function name is printed first in _alert api , so putting \n will not work properly , i tested